zorinos/zorinos-12.txt

```diff
@@ -0,0 +1,12 @@
+NAME="Zorin OS"
+VERSION="12"
+ID=Zorin OS
```

Comment:
Oh why nobody reads the specification? This field cannot have spaces in them.

NB: this is not your fault, thank you for contributing this patch